Integrations/MailSenderNew/MailSenderNew.py

```diff
@@ -301,28 +305,33 @@ def main():
     global SERVER
     FROM = demisto.getParam('from')
     FQDN = demisto.params().get('fqdn')
+    stderr_org = smtplib.stderr  # type: ignore
     try:
-        SERVER = SMTP(demisto.getParam('host'), int(demisto.getParam('port')), local_hostname=FQDN)
+        if demisto.command() == 'test-module':
```

Comment:
why are we triggering this only for test-module?

Reply:
for send-mail can affect memory